rules-reviewed/eap7/eap6/hibernate4.windup.xml

```diff
@@ -604,5 +572,160 @@
                 <matches pattern="(TableGenerator|TableHiLoGenerator|SequenceGenerator|SequenceIdentityGenerator|SequenceHiLoGenerator)" />
             </where>
         </rule>
+        <rule id="hibernate4-00029">
+            <when>
+                <javaclass references="org.hibernate.type.descriptor.sql.SqlTypeDescriptor">
```

Comment:
@mareknovotny - https://github.com/windup/windup/pull/858

This also requires that the rule use "implements_type" instead of inherits (since the sqlTypeDescriptor is an interface). And it still doesn't solve the issue of the correct jars not being on the library path during analysis.